WordPress/src/main/java/org/wordpress/android/ui/compose/components/buttons/PrimaryButton.kt

```diff
@@ -46,11 +70,17 @@ fun PrimaryButton(
             defaultElevation = 0.dp,
             pressedElevation = 0.dp,
         ),
+<<<<<<< HEAD:WordPress/src/main/java/org/wordpress/android/ui/compose/components/PrimaryButton.kt
+        colors = colors,
+        modifier = modifier
+            .padding(padding)
+=======
```

Comment:
Hey @thomashorta and @ovitrif 

Shamelessly pinging you since you both can resolve this conflict properly. I tried to resolve it but couldn't a way to handle for these two changes properly. 

I am sharing the commits so that you both can look at them and make sense 

- @thomashorta s changes - https://github.com/wordpress-mobile/WordPress-Android/commit/eebb64df4373c9876ac9d83fce9c96148fcdf7e6
- @ovitrif  s changes - https://github.com/wordpress-mobile/WordPress-Android/commit/9a8c283c59cc63766812aaccb832a7cef7c88179

Reply:
Regarding this, I opened a PR to manually merge both enhancements into one:
- https://github.com/wordpress-mobile/WordPress-Android/pull/18141